Packs/Campaign/Scripts/FindEmailCampaign/FindEmailCampaign.py

```diff
@@ -150,9 +150,12 @@ def create_context_for_campaign_details(campaign_found=False, incidents_df=None,
 
         incident_df.rename({FROM_DOMAIN_FIELD: 'emailfromdomain'}, axis=1, inplace=True)
         incidents_context = incident_df.fillna(1).to_dict(orient='records')
+        datetimes = incidents_df['created_dt'].dropna()  # type: ignore
```

Comment:
```suggestion
        datetimes: pandas.DataFrame = incidents_df['created_dt'].dropna()  # type: ignore
```
try something like this and only if this doesn't work please ignore only the specific error from mypy and not all of them.